src/test/java/com/chargerrobotics/unit/commands/autonomous/UT_AutoTurnStationary.java

```diff
@@ -0,0 +1,185 @@
+package com.chargerrobotics.unit.commands.autonomous;
+
+import static org.mockito.ArgumentMatchers.anyDouble;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+
+import com.chargerrobotics.commands.autonomous.AutoTurnStationary;
+import com.chargerrobotics.sensors.GyroscopeSerial;
+import com.chargerrobotics.subsystems.DriveSubsystem;
+import com.chargerrobotics.utils.SmartDashboardHelper;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class UT_AutoTurnStationary {
+
+    // Class to test with the unit test library
+    private AutoTurnStationary commandAutoTurnStationary;
+
+    // Classes that need to be mocked in order to support the command object
+    // functionality without having a real drive or gyro component
+    private  DriveSubsystem driveMock;
+    private  GyroscopeSerial gyroMock; 
+    private  SmartDashboardHelper smartDashboardMock;
+
+
+    @BeforeClass
+    public void setUp() {
+      
+      // Init unit under test
+      driveMock = mock(com.chargerrobotics.subsystems.DriveSubsystem.class);
+      gyroMock = mock(com.chargerrobotics.sensors.GyroscopeSerial.class);
+      smartDashboardMock = mock (com.chargerrobotics.utils.SmartDashboardHelper.class);
+
+    }
+    
+    
+  /**
+   * Test the 
+   *
+   * @param value The value returned by the physical controller
+   * @param hand The stick (left or right) to test against
+   * @param expectedValue The value that should be returned by as the reading
+   */
+  @Test()
+  public void testStartTurning90Degrees() {
+    // Reset mock to ensure valid data
+    reset(gyroMock);
```

Comment:
Since every test in this class does this reset work (and possibly a bit more), you can add it to a `@BeforeMethod` setup method to write this in one place.